src/client/stories/suggestedRoutes/suggestedRouteCard.js

```diff
@@ -0,0 +1,20 @@
+import React from "react";
+import SuggestedRouteCard from "../../components/SuggestedRoutes/SuggestedRouteCard";
+import { text, number } from "@storybook/addon-knobs";
+
+export function suggestedRouteCard() {
+  return (
+    <SuggestedRouteCard
+      time={number("Time", "26h / +6")}
+      price={number("Price", "$104.207/ -$6.208")}
+      eta={text("ETA", "2020-03-02 06:50:33")}
+      sailDays={number("Sail days", "6.5")}
+      maxWaveHs={number("Max wave HS", "11.52")}
+      avgSpeed={number("Average speed", "23.52")}
+      hireCost={number("Hire cost", "$66.808")}
+      consumption={number("Consumption", "$60.600")}
+    />
```

Comment:
Great work with the proptypes.

Now you just need to make sure that your stories are passing in all of the prop-types you declared. When I open storybook and open the console I see a lot of errors because here you are for example not passing in an ID or a `isSelected`prop.

Please go over this story as well as the other stories you've added and make sure all props are being fed in correctly :)

Reply:
That probably also means removing the units from the numbers above as we discussed, because you can't pass "$66.808" into a number knob for example.